app/questionnaire/placeholder_parser.py

```diff
@@ -46,9 +47,16 @@ def _resolve_value_source(self, value_source):
         if value_source["source"] == "metadata":
             return self._resolve_metadata_value(value_source["identifier"])
         if value_source["source"] == "list":
-            if value_source.get("id_selector") == "same_name_items":
-                return self._list_store[value_source["identifier"]].same_name_items
-            return len(self._list_store[value_source["identifier"]])
+            id_selector = value_source.get("id_selector")
```

Comment:
I didn't think we were updating runner for this?

Reply:
Yup, it requires a minor change it runner.